cylc/flow/rundb.py

```diff
@@ -493,8 +493,8 @@ def execute_queued_items(self):
                 raise
             self.n_tries += 1
             LOG.warning(
-                "%(file)s: write attempt (%(attempt)d) did not complete\n" % {
-                    "file": self.db_file_name, "attempt": self.n_tries})
+                "%(file)s: write attempt (%(attempt)d) did not complete: %(error)s\n" % {
+                    "file": self.db_file_name, "attempt": self.n_tries, "error": str(e)})
             if self.conn is not None:
```

Comment:
These lines just needed a little reformatting to fit within the project line length

```suggestion
            LOG.warning(
                "%(file)s: write attempt (%(attempt)d)"
                " did not complete: %(error)s\n" % {
                    "file": self.db_file_name,
                    "attempt": self.n_tries,
                    "error": str(e)
                }
            )
            if self.conn is not None:
```